9c697d609329ca10c7067a418bba1022/update/sys_rest_message_a671b6e1975512109790fefce053af17.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_rest_message">
+    <sys_rest_message action="INSERT_OR_UPDATE">
+        <access>package_private</access>
+        <authentication_type>no_authentication</authentication_type>
+        <basic_auth_password/>
+        <basic_auth_profile/>
+        <basic_auth_user/>
+        <description/>
+        <name>Plant disease import</name>
+        <oauth2_profile/>
+        <protocol_name/>
+        <rest_endpoint>https://perenual.com/api/pest-disease-list?key=sk-oq9f6710caae7048c7302</rest_endpoint>
```

Comment:
This contains your API Key - please remove it and make it a query parameter instead

Reply:
Hi @SapphicFire. I have removed the API key and used a query parameter instead.